src/utils/general.js

```diff
@@ -74,6 +75,13 @@ export const detailsToJSON = (values) => {
   if (values.uberType) {
     details.uberType = values.uberType;
   }
+  if (values.ratePerSec) {
```

Comment:
what do you think of this details to JSON checking each key like this?

i've had issues forgetting to add a check when doing a new proposal that needed a different details key.

maybe just something to consider when we do more proposal overhaul.